README.md

```diff
@@ -194,7 +194,7 @@ you can retrieve it by passing a symbol.
 
 ```ruby
 # app/policies/dashboard_policy.rb
-class DashboardPolicy < Struct.new(:user, :dashboard)
+class DashboardPolicy
```

Comment:
I'm not sure this is enough: to me this suggests that just defining an empty class with a `Policy` suffix is enough, while in reality we need a very specific initialiser and accessors to be defined. 

How would you feel about explicitly writing out the initialiser and accessors as part of this documentation?

Presumably the motivation behind using `Struct` was as a shorthand for that (but I do see why it doesn't work).

Alternatively maybe it's worth making a `HeadlessPolicy` class to help avoid this repeated boilerplate 🤷

Reply:
Another option is to use the other `Struct` approach, which Rubocop is OK with?

```ruby
DashboardPolicy = Struct.new(:user, :dashboard) do
  # ...
end
```

Reply:
True, I might also have come to that conclusion. Would this help to clarify? d17ca6c22279316e1cf65b58b351519f3797784d. As far as having an inherited class for headless policies, we actually just inherit from the same base policy class that all of our other policies use. The only difference is that the 2nd parameter of the initializer will just be the symbol passed to `authorize`, in this case `:dashboard`.